raft/src/main/java/org/apache/kafka/raft/LeaderState.java

```diff
@@ -301,6 +298,19 @@ public Endpoints leaderEndpoints() {
         return endpoints;
     }
 
+    int localId() {
+        return localReplicaKey.id();
+    }
```

Comment:
You can use `QuorumState.localIdOrThrow` since KRaft knows that this replica is the leader.

Reply:
thanks!